control-mvt/src/main/java/org/oskari/control/mvt/GetWFSVectorTileHandler.java

```diff
@@ -0,0 +1,270 @@
+package org.oskari.control.mvt;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.geotools.data.simple.SimpleFeatureCollection;
+import org.geotools.data.simple.SimpleFeatureIterator;
+import org.geotools.feature.DefaultFeatureCollection;
+import org.opengis.feature.simple.SimpleFeature;
+import org.oskari.service.mvt.SimpleFeaturesMVTEncoder;
+import org.oskari.service.mvt.wfs.TileCoord;
+import org.oskari.service.mvt.wfs.WFSMetaTiles;
+import org.oskari.service.mvt.wfs.WFSTileGrid;
+import org.oskari.service.util.ServiceFactory;
+
+import fi.nls.oskari.annotation.OskariActionRoute;
+import fi.nls.oskari.cache.Cache;
+import fi.nls.oskari.control.ActionConstants;
+import fi.nls.oskari.control.ActionException;
+import fi.nls.oskari.control.ActionHandler;
+import fi.nls.oskari.control.ActionParameters;
+import fi.nls.oskari.control.ActionParamsException;
+import fi.nls.oskari.domain.map.OskariLayer;
+import fi.nls.oskari.map.layer.OskariLayerService;
+import fi.nls.oskari.util.IOHelper;
+import fi.nls.oskari.util.ResponseHelper;
+
+@OskariActionRoute("GetWFSVectorTile")
```

Comment:
Add some comment about only supporting metric projections.

Reply:
It only supports EPSG:3067 (KNOWN_TILE_GRIDS) so far

Reply:
Nah, we just have to solve this in a smart way